app/views/layouts/merge_status.html.erb

```diff
@@ -4,7 +4,7 @@
     <%= stylesheet_link_tag *(params[:stylesheets] || []).select { |url| url.start_with?('https://assets-cdn.github.com/', 'https://github.githubassets.com/') } %>
     <%= stylesheet_link_tag 'merge_status' %>
   </head>
-  <body>
+  <body data-color-mode="<%= params[:mode] %>">
```

Comment:
How does it behave if this is empty?

Reply:
When `params[:mode]` is empty or missing from the params list the body will default to light mode so no changes to the current user experience.